### PR TITLE
A64FX, plus housekeeping and markdown support

### PR DESCRIPTION
**Pull request by heaton-dev**
Adding basic information for a64fx, bumping copyright, adding files to ignore list, add markdown support

### CODE REVIEW

File: user-guide/A64FX.rst [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
-Coming soon.
\ No newline at end of file
+HPE Apollo 80 cabinet, codenamed A64FX. Available since February 2021.
+
+* Fujitsu A64FX
+    * 48x Arm v8.2 SVE cores
+    * 32GB HBM2 memory arranged in 4x 12-core memory groups with 8GB each
+    * 64KB L1 cache, 8MB L2 cache per core memory group
+    * Mellanox Infiniband
+* Red Hat Enterprise Linux 8
```
Comment (christopheredsall) [inline user-guide/A64FX.rst:11]: should this be indented 4 spaces ?

Comment (heaton-dev) [reply user-guide/A64FX.rst:11]: No, the indented list are specifics to the Chip

Comment (heaton-dev) [reply user-guide/A64FX.rst:11]: Same on XCI page, but I had less to add to the list for now, we can add more later like the version and speeds of the Infiniband, specs of the SCRATCH storage, etc..

**Review by christopheredsall — APPROVED** [review]

File: user-guide/A64FX.rst (hunk 1 above)
Comment (christopheredsall) [reply user-guide/A64FX.rst:11]: Cool.

**Pull request merged**